doc/ErlangFormatterComparison.md

```diff
@@ -0,0 +1,345 @@
+# Erlang Formatter Comparison
+
+|                               |erlfmt                                                           |rebar3_format                |steamroller                                    |erl_tidy |
+|---                            |---                                                              |---                          |---                                            |--- |
+|[File Types](#file-types)                                  |.erl, .hrl, .app, .app.src, .config, .script, .escript           |.erl, .hrl	                |.erl, .hrl, .app, .app.src, .config, .script	|.erl |
+|[Macros](#macros)                                          |No crashes formatting OTP                                        |Skips entire files sometimes	|Skips entire files sometimes	                |Crashes sometimes |
+|[Comments](#comments)                                      |Preserves and moves to line before                               |Preserves but Floating       |Crashes sometimes and Reorders	                |Crashes sometimes and Floating |
+|[Configurable vs Opinionated](#configurable-vs-opinionated)|Opinionated                                                      |Configurable                 |Opinionated                                    |Configurable |
+|[Preserving Representation](#preserving-representation)    |Yes                                                              |Some                         |No                                             |No |
+|[Line Break Hints](#line-break-hints)                      |Yes                                                              |No                           |No                                             |No |
+|[Opt In/Out](#opt-inout)                                   |per file, per top level expression                               |No                           |No                                             |No |
```

Comment:
You can opt out in `rebar3_format`, just add `-format ignore.` to your file and it won't be formatted.